axon-reactor/src/main/java/org/axonframework/extensions/reactor/commandhandling/gateway/DefaultReactorCommandGateway.java

```diff
@@ -126,6 +124,10 @@ public Registration registerResultHandlerInterceptor(
                 .map(it -> (R) it.getPayload());
     }
 
+    private Mono<? extends CommandResultMessage<?>> mapExceptionalResult(CommandResultMessage<?> response) {
```

Comment:
Wow, interesting. But for Commands I did get it to work with an `onErrorResume`. How come this is still necessary?